README.md

```diff
@@ -155,36 +101,3 @@ Set your presentation subclass to the KetchUI instance:
 `ketchUI.sizeFactory = ExampleSizeFactory()`
   
 </details>
-
-
-## Running the Sample app
```

Comment:
lets add a link to the new location of sample app
we may still need a reference to Sample app here, so that the developers could explore some live examples